Filtration/ViewModels/ItemFilterCommentBlockViewModel.cs

```diff
@@ -10,12 +11,34 @@ internal interface IItemFilterCommentBlockViewModel : IItemFilterBlockViewModelB
 
     internal class ItemFilterCommentBlockViewModel : ItemFilterBlockViewModelBase, IItemFilterCommentBlockViewModel
```

Comment:
Duplicating all of these commands in this class isn't the right way to fix this - they should be moved to `ItemFilterBlockViewModelBase` as both `ItemFilterCommentBlockViewModel` and `ItemFilterBlockViewModel` derive from this class (they'll also need moving from the `IItemFilterBlockViewModel` interface to `IItemFilterBlockViewModelBase` too). This was an oversight on my part in the refactoring of comment handling (the commits I made in May/June that were included in 0.16).

Reply:
I'm not 100% sure what you want transfered from ItemFilterBlockViewModel to IItemFilterBlockViewModel.

Reply:
Err, never mind about that part, I thought the RelayCommands were also in the interface but they aren't.